javascript.js

```diff
@@ -23,3 +25,270 @@ document.querySelectorAll('.nav-item').forEach((n) => n.addEventListener('click'
+main_card_tag_item.textContent = main_project.features[0];
+main_card_tags.appendChild(main_card_tag_item);
+main_card_tag_item = document.createElement('li');
+main_card_tag_item.textContent = main_project.features[1];
+main_card_tags.appendChild(main_card_tag_item);
+main_card_tag_item = document.createElement('li');
+main_card_tag_item.textContent = main_project.features[2];
+main_card_tags.appendChild(main_card_tag_item);
+
+main_card_content.appendChild(main_card_h3);
+main_card_content.appendChild(main_card_p);
+main_card_content.appendChild(main_card_tags);
+main_card_content.appendChild(main_card_button);
+
+const main_project_img = document.createElement('img');
+main_project_img.id = 'featured_img';
+main_project_img.setAttribute('src', 'images/Img_Placeholder.png');
+
+main_project_container.appendChild(main_card_content);
+main_project_container.appendChild(main_project_img);
+
+// main card pop up action
+main_card_button.addEventListener('click', () => {
+  const popup = document.createElement('div');
+  popup.className = 'popup_container';
+  popup.classList.add('text_styles');
+
+  const popup_header = document.createElement('div');
+  popup_header.className = 'popup_header';
+  popup_header.appendChild(main_card_h3);
+  popup_header.appendChild(main_card_tags);
+
+  const popup_img = document.createElement('img');
+  popup_img.setAttribute('src', 'images/Img_Placeholder.png');
+
+  const live_button = document.createElement('a');
+  live_button.textContent = ' See Live';
+  live_button.className = 'project-button';
+
+  const source_button = document.createElement('a');
+  source_button.textContent = 'See Source';
+  source_button.className = 'project-button';
+
+  const see_live_icon = document.createElement('img');
+  see_live_icon.setAttribute('src', 'images/Icon-see live.svg'); see_live_icon.setAttribute('style', 'width: 16px; margin-left: 10px');
+  const source_icon = document.createElement('img');
+  source_icon.setAttribute('src', 'images/Git-Hub.svg'); source_icon.setAttribute('style', 'width: 16px; margin-left: 10px');
+
+  live_button.appendChild(see_live_icon);
+  source_button.appendChild(source_icon);
+
+  const popup_info = document.createElement('div');
+  popup_info.className = 'popup_info';
+  popup_info.appendChild(main_card_p);
+  popup_info.appendChild(live_button);
+  popup_info.appendChild(source_button);
+
+  const close_button = document.createElement('img');
+  close_button.setAttribute('src', 'images/CloseBtn_dark.svg');
+  close_button.id = 'close_btn';
+
+  popup.appendChild(close_button);
+  popup.appendChild(popup_info);
+  popup.appendChild(popup_header);
+  popup.appendChild(popup_img);
+
+  document.body.appendChild(popup);
+  addBlur();
+
+  close_button.addEventListener('click', () => {
+    document.body.removeChild(popup);
+    main_card_content.appendChild(main_card_h3);
+    main_card_content.appendChild(main_card_p);
+    main_card_content.appendChild(main_card_tags);
```

Comment:
- [ ] it would be better to use the size font-family for tags in order to match the Figma design please make sure to do that 

<img width="529" alt="Screenshot 2023-03-09 at 12 28 14" src="https://user-images.githubusercontent.com/61790587/224010114-64fd9d0f-8f2f-4e41-ab91-310b7f6d0104.png">